### ISSUE TITLE
fix(jans-auth): super_gluu login doesn't work with jans server

### ISSUE THREAD

**Issue by imShakil** [JanssenProject/jans#4132]
I tried SSO flow using mod_oauth_oidc to test super gluu auth with jans server. It's failed to authenticate. 

![IMG_0621 2](https://user-images.githubusercontent.com/20867846/224878717-81faaa80-739e-4ece-945b-be27a75d0899.jpg)

Steps to reproduce:
1. Install jans server and letsencrypt certificate
2. enable super_gluu script in jans server
3. create a client id and set the default auth as super_gluu at jans server
4. create rp with mod_oauth_oidc
5. setup sso
6. test super gluu login

device & version:

- Jans server (1.0.9)
- RP (mod_oauth_oidc)
- super gluu (ios & android) v4.4
logs: [jans-auth.log](https://github.com/JanssenProject/jans/files/11084161/jans-auth.log)

**Comment by nynymike** [comment]
I'm not sure what you mean by Step 3: "create an client id and set default auth as super_gluu at jans server" -- did you create this on the server that needs to call oxnotify (api.gluu.org)?

**Comment by imShakil** [comment]
not really. I meant to create a client with default acr which is actually needed in `mod_oauth_oidc` setup.
